modules/overviews/app/components/overviews/widgets/news_component.rb

```diff
@@ -54,6 +54,10 @@ def initialize(*)
       def title
         Project.human_attribute_name(:news)
       end
+
+      def render?
+        project.nil? || project.module_enabled?("news")
```

Comment:
A compromise might be to move the workspace type check to the actual component:

```suggestion
        project.nil? || project.project? && project.module_enabled?("news")
```

Reply:
I don't agree with that.  Imho, the news widget should not care about which workspace type it is rendered in.. Especially now, where we have individual components per workspace which can take of the widgets they want to render.

Reply:
@HDinger yeah, you're right. I prefer the individual components per workspace approach.